db/migrate/20230920100700_add_details_to_comments.rb

```diff
@@ -0,0 +1,9 @@
+class AddDetailsToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :user_id, :bigint
+    add_index :comments, :user_id
+    add_column :comments, :post_id, :bigint
+    add_index :comments, :post_id
+    add_column :comments, :text, :text
+  end
```

Comment:
- [Optional] Dear @sunga12, I see you created a table without a column and applied a `change` method to add the columns. Usually `change` method is used to add a column that is missed or some other changes to the table that should be applied. Please consider creating a table with all its columns.

Reply:
Noted thanks. will do going forward